build/import/Versions.props

```diff
@@ -1,28 +1,17 @@
 <!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information. -->
-<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <!-- IMPORTANT: These assembly versions must always be 3 digits -->
-    <MinimumRequiredVSVersion>16.8.1</MinimumRequiredVSVersion> <!-- We should try to keep this up to date with the highest version available on build machines -->
-    <ProjectSystemVersion>17.3.0</ProjectSystemVersion> <!-- This should always match with the VS release we're targeting -->
-    <VersionBase>$(ProjectSystemVersion)</VersionBase>
-    <PreReleaseVersionLabel>preview3</PreReleaseVersionLabel>
-    <SuppressNETCoreSdkPreviewMessage>true</SuppressNETCoreSdkPreviewMessage>
-
-    <!-- Opt-in and out of repo features -->
-    <UsingToolVSSDK>true</UsingToolVSSDK>
-    <UsingToolIbcOptimization>true</UsingToolIbcOptimization>
-    <UsingToolXUnit>false</UsingToolXUnit>
-    <UsingToolMicrosoftNetCompilers>false</UsingToolMicrosoftNetCompilers>
-
-    <!-- Toolset -->
-    <RoslynToolsRepoToolsetVersion>1.0.0-beta-62705-01</RoslynToolsRepoToolsetVersion>
-    <RoslynToolsModifyVsixManifestVersion>1.0.0-beta-62327-02</RoslynToolsModifyVsixManifestVersion>
-    <RoslynToolsSignToolVersion>1.1.0-beta3.21260.1</RoslynToolsSignToolVersion>
-    <MicrosoftDiaSymReaderPdb2PdbVersion>1.1.0-beta1-62624-01</MicrosoftDiaSymReaderPdb2PdbVersion>
-    <MicroBuildPluginsSwixBuildVersion>1.1.37</MicroBuildPluginsSwixBuildVersion>
-    <CodecovVersion>1.12.4</CodecovVersion>
```

Comment:
Specifying package versions in this file was only necessary for very specific scenarios. Most of these packages are no longer used. The ones that are still used are now generating a path property, which is the only reason the version number was needed outside of a `PackageReference` node.